subgraph-js/src/__tests__/buildSubgraphSchema.test.ts

```diff
@@ -454,9 +454,9 @@ extend interface Product @key(fields: "upc") {
         }
       `);
 
-      const { data, errors } = await graphql(schema, query);
+      const { data, errors } = await graphql({ schema, source: query });
```

Comment:
This is interesting to me that we can't specify the expected type of `data` via a type argument. @IvanGoncharov is there a reason it doesn't?

Reply:
@trevor-scheer Since `graphql` can't ensure that type is valid it would work exactly the same as `as SomeType`. Moreover, directives can change the shape of response (e.g. upcoming `@stream/defer`).
I recommend just doing explicit type cast here.